source/_components/light.limitlessled.markdown

```diff
@@ -80,3 +80,129 @@ Refer to the [light]({{site_root}}/components/light/) documentation for general
+
+### Link/bind a bulb
+
+Before you can use HomeAssistant to control your bulbs you have to pair them to the phone app. This sends the config to the bridge. Once linked you don't have to use the MiLight app at all if you don't want to.
+
+Pair your bulbs to the relevant 4 zones (tabs) along the top. These are the four groups that map to the 'numbers' in the configuration.yaml. For example the first tab or zone in the App is '- number: 1' in configuration.yaml.
+
+Make sure you choose the relevant logical remote control in the app for your lights. I messed around for hours and hours trying to use the default remote that had colour and colour temp. 
+
+1. once the bridge is connected to your phone and network
+1. open MiLight 3.7
+1. click on the MiLight device
+1. you will see the 4 zones along the top
+1. click the two arrows icon in the top right to choose the relevant remote type for your bulb 
+1. back on the main page, click the chainlink icon in top right corner to get to the Link/Unlink page
+1. unlink the bulb first (if you have already used it before)
+   1. turn the light you want to connect to off and back on again at the wall
+   1. within 3 secs press the red Unlink button
+   1. light will flash 9 times and will now be unlinked
+1. select the tab/zone along the top for the zone you want to put the bulb into
+1. turn the light you want to connect to off and back on again at the wall
+1. within 3 secs press the green Link button
+1. light will flash 4? times and will now be linked
+1. go back to the main page and you can control it.
+1. Repeat the linking process for each bulb you want connected to the same group, ie. 4x kitchen lights all connected to the same tab.
+
+I Have:
+- 4x GU10 ww/cw bulbs for the kitchen spotlight fixture, assigned to zone 1 on the app using the 10th remote control along from the left; it's the white only with 4 blue groups buttons at the bottom.
+- The remaining bulbs are all E27 RGBW added to the relevant zone using the 2nd remote control from the left in the MiLight app v 3.7; this is the colour-only remote with 4 yellow group buttons at the bottom.
+  - Kitchen table, also added to zone 1, but from the colour only remote stated above. This appears to allow me to independently control it even though it's on the same zone as the 4x GU10 bulbs. This is because they're of a different 'type', 'white' vs 'rgbw' (see the config below). This is explained in [LimitlessLED example page](https://home-assistant.io/components/light.limitlessled/) 
+  - Lounge & Hall, both are bound to zone 2
+  - Study, bound to zone 3
+  - Bedroom, bound to zone 4
+
+Other Tips I found on an article on the LimitlessLED website that might be of use:
+
+We also had some feedback/tips from users:
+1. Tip#10. I had to use the LimitlessLED v1.7 app to connect the bridge to my LAN, then it worked, thanks.
```

Comment:
**Spelling & Grammar**:
- Consider number the list correctly, this improves readability of the Markdown file.
- Concern: Why are the tips numbered from 10 and upwards?